core/src/main/java/com/rim/IceField/Map.java

```diff
@@ -29,6 +29,7 @@ public Map() {
     }
 
 
+
     //Method for generating icebergs and items on the map.
     public void generateItemsOnMap() {
 
```

Comment:
I guess for now we could live this test, but when we were planing this test, I was thinking about how well our map would be generated. Let's see, maybe we could add another test for that purpose;